README.md

```diff
@@ -36,6 +36,9 @@ The script will:
 ### --quiet
 (Optional) Disable debug output, only the resulting changelog will be printed.
 
+### --prefixes, -pf
```

Comment:
Let's change the alias to single-letter (-R?) because multile letters might break the logic when using short arguments notation like `command -rfabc`, or maybe remove alias at all

Reply:
I removed alias for this command.  I think this is the best option in this case.